website/static/js/osfHelpers.js

```diff
@@ -701,7 +701,22 @@ var confirmDangerousAction = function (options) {
 
     bootbox.dialog(bootboxOptions);
 };
-
+/**
+ * Maps an object to an array of {key: KEY, value: VALUE} pairs
+ *
+ * @param {Object} obj
+ * @returns {Array} array of key, value pairs
+ **/
+var iterObject = function(obj) {
```

Comment:
I can live with that. @lyndsysimon removed/deactivated a bunch of code while separating comments from drafts- you may want to ping him to verify that his branch isn't missing something required by a mako template.